src/index.html

```diff
@@ -11,7 +11,136 @@
+        <label class="form_field">
+          Surname:
+          <input 
+            type="text"
+            name="Surname"
+          >
+        </label>
+        <label class="form_field">
+           Name:
+          <input 
+            type="text"
+            name="Name"
+          >
+        </label>
+        <label class="form_field">
+          How old are You?
+          <input 
+            type="number"
+            name="quantity"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form_field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthday"
+          >
+        </label>
+        <label class="form_field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form_field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field">
+          Do you love cats?
+          <input type="radio"
+            name="cat"
+            value="Yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="cat"
+            value="No"
+          >
+          No
+        </label>
+        <label class="form_field">
+          What is your favorite color?
+          <input type="color" name="favcolor">
+        </label>
+        <label class="form_field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            step="2"
+          >
+        </label>
+        <label class="form_field">
+          What are your faforite brand of cars?
+          <select name="car" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form_field">
+          How do you rate our work?
+          <input
+            type="range"
+            id="vol"
+            name="vol"
+            min="0"
+            max="20"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form">
+        <legend> Addititional info</legend>
+        <label class="form_field">
+          Comments
+          <textarea name="comment"
+            rows="2"
+            cols="20"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent this
<img width="333" alt="image" src="https://user-images.githubusercontent.com/53611910/205504086-9aec1c88-b6db-4aca-8684-8194465da340.png">